docs/deploy-project.sh

```diff
@@ -59,8 +64,15 @@ function deploy() {
 }
 
 function merge() {
+    # Specify consumers configuration with the default configuration in the format:
```

Comment:
are these lines necessary here while they are already in the project-base? This is for the scenario where a project does not have its own configurations, right? (This should not be a case for the current commerce cloud/platform projects, but might be a case for older pure Framework projects, am I right?)